sdk/src/Core/Amazon.Runtime/Internal/InternalConfiguration.cs

```diff
@@ -285,6 +299,18 @@ private string GetClientAppIdEnvironmentVariable()
 
             return rawValue;
         }
+
+        /// <summary>
+        /// Gets a simple string value from an environment variable.
+        /// </summary>
+        private string GetStringEnvironmentVariable(string name)
```

Comment:
Does the SEP say anything about trimming the value? There's already a `GetEnvironmentVariable<T>`  method that could be re-used (`T` for the new variable would be `string`).